aws_xray_sdk/core/emitters/udp_emitter.py

```diff
@@ -32,12 +32,15 @@ def send_entity(self, entity):
 
         :param entity: a trace entity to send to the X-Ray daemon
         """
-        message = "%s%s%s" % (PROTOCOL_HEADER,
-                              PROTOCOL_DELIMITER,
-                              entity.serialize())
+        try:
```

Comment:
The `try-except` block should be tighter if we really just want to skip sending data to daemon in case of serialization failure. With the current try-except block and the log message, we won't know if the exception was in entity serialization or in sending data to daemon.

Reply:
The `log.exception` will print out the trace stack so we can know where it goes wrong, either during serialization or sending data to daemon.

Reply:
Oh yes. Right.